src/byrow/doc.jl

```diff
@@ -679,7 +679,7 @@ julia> byrow(ds, select, :, with = byrow(ds, findfirst, :, by = isodd))
  7
 ```
 @@@@fill!@@@@
-    byrow(ds::AbstractDataset, fill!, cols; [with, by = ismissing, rolling = false, threads])
+    byrow(ds::Dataset, fill!, cols; [with, by = ismissing, rolling = false, threads])
```

Comment:
please leave this as AbstractDataset, because, `byrow(fill!)` can fill view of Dataset.